src/actions/guides/database/models.cr

```diff
@@ -7,70 +8,160 @@ class Guides::Database::Models < GuideAction
     * `Int32` - `int` column type.
-    * `Time` - `timestamp` column type.
-    * `Float64` - `decimal` column type.
-    * `Bool` - `bool` column type.
+    * `Int64` - `bigint` column type.
+    * `Float` - `decimal` column type.
+    * `Bool` - `boolean` column type.
+    * `Time` - `timestamptz` column type.
+    * `UUID` - `uuid` column type.
+
+    Any of your columns can also define "nillable" types by adding Crystal `Nil` Union `?`. This is if your column allows for a `NULL` value. (e.g. `column age : Int32?` allows an `int` or `NULL` value).
+
+    ## Model associations
 
-    ## Using associations
+    In a [RDBMS](https://en.wikipedia.org/wiki/Relational_database) you may have tables that are related to each other. With Avram, we can associate two models to make some common queries a lot more simple.
 
-    Right now you can define `has_many`, `has_one`, and `belongs_to` associations.
+    All associations will be defined in the `table` block. You can use `has_one`, `has_many`, and `belongs_to`.
+
+    To avoid running in to an "Undefined Constant" error, be sure to require each model you want to associate.
 
     ```crystal
-    # Require the models we want to associate. If we don't we may get an
-    # "Undefined constant" error because Lucky didn't load the associated models yet
     require "./supervisor"
     require "./task"
     require "./company"
-
     class User < BaseModel
-      table :users do
+      table :user do
         has_one supervisor : Supervisor
         has_many tasks : Task
         belongs_to company : Company
       end
     end
+    ```
+
+    ## Belongs to
+
+    A `belongs_to` will assume you have a foreign key column related to the other model defined as `{model_name}_id`.
+
+    ```crystal
+    table :users do
+      column name : String
 
+      # gives you the `company_id`, and `company` methods
+      belongs_to company : Company
+    end
+    ```
+
+    > When you create the migration, be sure you've set `[add_belongs_to](#{Guides::Database::ManagingAndMigrating.path})`.
+
+    You can preload these associations in your queries, and return the associated model.
+
+    ```crystal
     # Will return the company associated with the User
     UserQuery.new.preload_company.find(1).company
     ```
 
-    ### Making associations optional
+    ### Optional association
 
     Sometimes associations are not required. To do that add a `?` to the end of the type.
 
-    ```crystal
+     ```crystal
     belongs_to company : Company?
     ```
 
-    > Make sure to make the column nilable in your migration as well: `belongs_to Company?`
+    > Make sure to make the column nilable in your migration as well: `add_belongs_to company : Company?`
 
-    ### Has many through (many-to-many)
+    ## Has one (one to one)
+
+    ```crystal
+    table :users do
+      has_one supervisor : Supervisor
+    end
```

Comment:
I believe this should also show the other direction (`belongs_to`) so it is clear how the relationship works.